lamindb_setup/core/_settings_instance.py

```diff
@@ -333,6 +333,10 @@ def _check_sqlite_lock(self):
     @property
     def db(self) -> str:
         """Database connection string (URI)."""
+        if "LAMINDB_DATABASE_URL" in os.environ:
```

Comment:
Shouldn't this be called `LAMINDB_INSTANCE_DB`, because that's what it's called in pydantic, which has the same behavior.

Reply:
Uh, this is yet another env varibale here https://github.com/laminlabs/lamindb-setup/blob/db0d9a0b8121e188a0be01288bde7f399ab83d97/lamindb_setup/_connect_instance.py#L73 i wasn't aware

Reply:
Then need to rework this a bit.